earth_enterprise/src/SConstruct

```diff
@@ -367,6 +367,37 @@ installdirs = {
     'htdocs': InstServerDir('gehttpd/htdocs'),
 }
 
+def get_cc_version(cc_path):
+  """Extracts the version number of a C/C++ compiler (like GCC) as a list of
+  strings."""
+
+  process = subprocess.Popen(
+    [cc_path, '--version'], stdout=subprocess.PIPE)
+  (stdout, stderr) = process.communicate()
+  match = re.search('[0-9][0-9.]*', stdout)
+  # If match is None we don't know the version.
+  if match is None:
+    return None
+  version = match.group().split('.')
+
+  return version
+
+
+def get_cc_version(cc_path):
```

Comment:
This looks like a duplicate of the function above, rather than a definition of the `is_version_ge` function.